### PR TITLE
Added parallel-by-default config

### PR DESCRIPTION
**Pull request by SloEddy**

### CODE REVIEW

File: config/php-formatter.php [hunk 1]
```diff
@@ -89,6 +89,9 @@
         'trim_array_spaces' => true,
         'whitespace_after_comma_in_array' => true,
     ],
+    // You can initialises your own ParallelConfig class here with your desired parameters
+    // default is set to detect the environment and use the appropriate resources
+    // 'parallel' => PhpCsFixer\Runner\Parallel\ParallelConfigFactory::detect(),
```
Comment (PauloMoreiraCL) [inline config/php-formatter.php:94]: should this line be commented?

**Review by PauloMoreiraCL — COMMENTED** [review]
Just a small question

File: config/php-formatter.php (hunk 1 above)
Comment (SloEddy) [reply config/php-formatter.php:94]: yep, by default we can just leave it 'undefined' which will end up using the default detection https://github.com/ComLaude/php-formatter/pull/4/files#diff-a7f7f4b6ef0ee4ae3d05932f2a7109985e234ceee3b84c24060b487dcb403983R46

I just wanted to make sure it's clear in the readme how you use this (and the config file is often what people look at when deciding what to change)

**Review by PauloMoreiraCL — APPROVED** [review]
Linter is way faster now, awesome!

**Pull request merged**